smokes/e2e/pages/force.ts

```diff
@@ -49,6 +49,17 @@ export class ForcePage extends BasePage {
         await button.click();
     }
 
+    async clickStartButtonAndGoToBuild() {
```

Comment:
this suggest we force a go to the build.

```suggestion
    async clickStartButtonAndWaitRedirectToBuild() {
```

Reply:
Fixed.